lib/screen/detail/artwork_detail_page.dart

```diff
@@ -544,23 +544,24 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
             },
           ),
         ],
-        OptionItem(
-          title: 'view_on_'.tr(args: [asset.secondaryMarketName]),
-          icon: SvgPicture.asset(
-            'assets/images/external_link.svg',
-            width: 20,
-            height: 20,
+        if (asset.secondaryMarketURL.isNotEmpty)
```

Comment:
please use the isValid function. If the link is invalid, don't remove this option, but disable it.

Reply:
Since we create the url ourself, either it is empty or it is valid